.travis.yml

```diff
@@ -0,0 +1,18 @@
+sudo: required
+
+language: minimal
+
+services: docker
+
+branches:
+    only:
+        - master
```

Comment:
We should start with something like https://github.com/libero/php-coding-standard/blob/0e12fdeb2b97d826d7c92b290d946a6e62fd463c/.travis.yml#L45-L49.

Reply:
Updated, with a regex that tries to be immediately understandable